specification/peering/resource-manager/Microsoft.Peering/preview/2020-01-01-preview/peering.json

```diff
@@ -0,0 +1,3271 @@
+          "description": "The properties that define an exchange peering."
+        },
+        "peeringLocation": {
+          "description": "The location of the peering.",
+          "type": "string"
+        },
+        "provisioningState": {
+          "description": "The provisioning state of the resource.",
+          "enum": [
+            "Succeeded",
+            "Updating",
+            "Deleting",
+            "Failed"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "provisioningState",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "PeeringPropertiesDirect": {
+      "description": "The properties that define a direct peering.",
+      "type": "object",
+      "properties": {
+        "connections": {
+          "description": "The set of connections that constitute a direct peering.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DirectConnection"
+          }
+        },
+        "useForPeeringService": {
+          "description": "The flag that indicates whether or not the peering is used for peering service.",
+          "type": "boolean",
+          "readOnly": true
+        },
+        "peerAsn": {
+          "$ref": "#/definitions/SubResource",
+          "description": "The reference of the peer ASN."
+        },
+        "directPeeringType": {
+          "description": "The type of direct peering.",
+          "enum": [
+            "Edge",
+            "Transit",
+            "Cdn",
+            "Internal",
+            "Ix",
+            "IxRs"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "directPeeringType",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "PeeringPropertiesExchange": {
+      "description": "The properties that define an exchange peering.",
+      "type": "object",
+      "properties": {
+        "connections": {
+          "description": "The set of connections that constitute an exchange peering.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ExchangeConnection"
+          }
+        },
+        "peerAsn": {
+          "$ref": "#/definitions/SubResource",
+          "description": "The reference of the peer ASN."
+        }
+      }
+    },
+    "DirectConnection": {
+      "description": "The properties that define a direct connection.",
+      "type": "object",
+      "properties": {
+        "bandwidthInMbps": {
+          "format": "int32",
+          "description": "The bandwidth of the connection.",
+          "type": "integer"
+        },
+        "provisionedBandwidthInMbps": {
+          "format": "int32",
+          "description": "The bandwidth that is actually provisioned.",
+          "type": "integer",
+          "readOnly": true
+        },
+        "sessionAddressProvider": {
+          "description": "The
```

Comment:
I don't find the usage of this definition. Do we need this one?

Reply:
removed.